supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfigurationAttributes.scala

```diff
@@ -202,7 +204,8 @@ object PipelinesApiConfigurationAttributes {
     }
 
     (project, executionBucket, endpointUrl, genomicsAuthName, genomicsRestrictMetadataAccess, gcsFilesystemAuthName,
-      qpsValidation, duplicationStrategy, requestWorkers, localizationConfiguration, virtualPrivateCloudConfiguration, batchRequestTimeoutConfigurationValidation) flatMapN authGoogleConfigForPapiConfigurationAttributes match {
+      qpsValidation, duplicationStrategy, requestWorkers, localizationConfiguration, virtualPrivateCloudConfiguration,
+      batchRequestTimeoutConfigurationValidation) flatMapN authGoogleConfigForPapiConfigurationAttributes match {
```

Comment:
TOL (my personal preference) if you split one of these onto a new line, I prefer if we split the entire list onto newlines

Reply:
I have split the entire list by newlines 👍